Doberman/Pipeline.py

```diff
@@ -71,8 +71,9 @@ def process_cycle(self):
         doc = self.db.get_pipeline(self.name)
         sensor_docs = {n: self.db.get_sensor_setting(n) for n in self.depends_on}
         self.reconfigure(doc['node_config'], sensor_docs)
-        status = 'silent' if self.cycles <= self.startup_cycles else doc['status']
-        if status != 'silent':
+        is_silent = (self.cycles <= self.startup_cycles) or (doc['silent_until'] > time.time()) or \
```

Comment:
Explicitly use UTC time? Or is it timezone aware? Would be good if this can run on systems that have a different timezone set

Reply:
`utils.dtnow?`

Reply:
time.time() creates a unix timestamp which is timezone independent. The only benefit I see of storing a datetime object rather than the timestamp is for better readability in the database.

Reply:
Perfect, I am happy with a timestamp :)